File: Plugins/engines/org.eclipse.escriptmonkey.scripting.engine.javascript.rhino.samples/samples/scripts/IO__File_Sample.js
/*
 * Menu: Examples > IO > JS >File Sample
 * Kudos: Paul Colton, Aptana, Inc.
 * License: EPL 1.0
 * Description : You can use this script to display into a dialog box the current date.
 * DOM: http://download.eclipse.org/technology/dash/update/org.eclipse.eclipsemonkey.lang.javascript
 */
  
function main() 
{
	var date = new Date();

	// Write out new file
	var file = new File("myFile.txt");
	file.createNewFile();
	file.write("Date: " + date);
	
	// Read lines from file
	var text = file.readLines();
	
	// Display what we read
	Packages.org.eclipse.jface.dialogs.MessageDialog.openInformation( 	
		window.getShell(), 	
		"File I/O", 
		"Contents: " + text	
	);
	
}

File: Plugins/engines/org.eclipse.escriptmonkey.scripting.engine.python.jython.samples/samples/scripts/IO__File_Sample.py
#
# Menu: Examples > IO > Py > File Sample
# Kudos: Paul Colton, Aptana, Inc.
# Description : You can use this script to display into a dialog box the current date.
# License: EPL 1.0
#

import time

import org.eclipse.jface.dialogs

date = time.strftime("%Y-%m-%d %H:%M:%S")

# Write out new file
file = open("myFile.txt", "wt")
file.write("Date: " + date)
file.close()

# Read lines from file
file = open("myFile.txt", "rt")
text = file.read()
file.close()

# Display what we read
org.eclipse.jface.dialogs.MessageDialog.openInformation( 	
	window.getShell(), 	
	"File I/O", 
	"Contents: " + text	
)
